srcpkgs/libinput-gestures/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'libinput-gestures'
+pkgname=libinput-gestures
+version=2.38
+revision=1
+noarch=yes
+depends="python3 libinput xdotool wmctrl hicolor-icon-theme"
+short_desc="Action gestures on your touchpad using libinput"
+maintainer="Chinmay Pai <chinmaydpai@gmail.com>"
+license=" GPL-3.0-or-later"
+homepage="https://github.com/bulletmark/${pkgname}"
+distfiles="${homepage}/archive/${version}.tar.gz"
+conf_files="/etc/libinput-gestures.conf"
```

Comment:
conf_files should be directly below distfiles=

Reply:
wait what

Reply:
ah sorry saw it wrong.